### PR TITLE
Basic pep 8 corrections, bug fix in string formatting. 

### PR DESCRIPTION
**Pull request by razmikmelikbekyan**
*Issue #, if available:*

*Description of changes:*

Basic pep 8 corrections:
 - imports optimisation - https://www.python.org/dev/peps/pep-0008/#imports
 - surround top-level function and class definitions with two blank lines - https://www.python.org/dev/peps/pep-0008/#blank-lines
Bug fix in string formating.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autogluon/task/base/base_task.py [hunk 1]
```diff
@@ -66,6 +70,7 @@ def run_fit(cls, train_fn, search_strategy, scheduler_options):
                        args=record_args)
         return results
 
+    @classmethod
     @abstractmethod
```
Comment (Innixma) [inline autogluon/task/base/base_task.py]: Shouldn't @abstractmethod be removed if it is a @classmethod?

Comment (razmikmelikbekyan) [reply autogluon/task/base/base_task.py]: It is not obligatory, they can stay together. This means that `fit` method is a `abstractmethod` so all children must implement it and on the other hand with `classmethod` you indicate that it children should implement it as a `classmethod`. See the reference : https://docs.python.org/3.6/library/abc.html#abc.abstractmethod

**Comment by szha** [comment]
Job PR-279-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-279/1/index.html

File: autogluon/task/base/base_task.py (hunk 1 above)
Comment (Innixma) [reply autogluon/task/base/base_task.py]: I see, sorry I misread abstractmethod to mean staticmethod. Thanks for the explaination!

**Pull request merged**